server/utils/referralUtils.test.ts

```diff
@@ -60,25 +60,23 @@ describe('ReferralUtils', () => {
   })
 
   describe('isReadyForSubmission', () => {
-    const referral = referralFactory.build({ reason: undefined })
-
     it('returns false for a new referral', () => {
-      expect(ReferralUtils.isReadyForSubmission(referral)).toEqual(false)
+      const newReferral = referralFactory.started().build()
+
+      expect(ReferralUtils.isReadyForSubmission(newReferral)).toEqual(false)
     })
 
-    describe('when OASys is confirmed and a reason is provided', () => {
+    describe('returns true when OASys is confirmed and a reason is provided', () => {
```

Comment:
have we lost an `it` block here or did you mean to update this?

Reply:
Ah, I meant to make this a denested `it` block, like the test above. There's no happy/sad path here, just multiple conditions, so it didn't feel like one should be nested and the other not. Are you happy with both being denested (`it`s)?

Reply:
Yep, happy with the `it`s, I thought it was probably a typo, just wanted to make sure I hadn't missed some new style of using `describe`s!